From a4b080c130c03877db2a51105b59e4092056984d Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Mon, 29 Jun 2026 22:10:02 +0200
Subject: [PATCH] Enable -fipa-pta by default for -Ofast

---
 gcc/opts.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gcc/opts.cc b/gcc/opts.cc
index 6658b6acd..ea3894a1c 100644
--- a/gcc/opts.cc
+++ b/gcc/opts.cc
@@ -725,6 +725,7 @@ static const struct default_options default_options_table[] =
     { OPT_LEVELS_FAST, OPT_ffast_math, NULL, 1 },
     { OPT_LEVELS_FAST, OPT_fallow_store_data_races, NULL, 1 },
     { OPT_LEVELS_FAST, OPT_fsemantic_interposition, NULL, 0 },
+    { OPT_LEVELS_FAST, OPT_fipa_pta, NULL, 1 },
 
     { OPT_LEVELS_NONE, 0, NULL, 0 }
   };
-- 
2.36.6

